homeassistant/components/steam_online/config_flow.py

```diff
@@ -0,0 +1,222 @@
+                return self.async_abort(reason="already_configured")
+        _msg = "Steam yaml config in now deprecated and has been imported. Please remove it from your config"
+        LOGGER.warning(_msg)
+        import_config[CONF_ACCOUNT] = import_config[CONF_ACCOUNTS][0]
+        return await self.async_step_user(import_config)
+
+    async def async_step_reauth(self, user_input: dict[str, str]) -> FlowResult:
+        """Handle a reauthorization flow request."""
+        self.entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
+
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Confirm reauth dialog."""
+        if user_input is not None:
+            return await self.async_step_user()
+
+        self._set_confirm_only()
+        return self.async_show_form(step_id="reauth_confirm")
+
+
+class SteamOptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle Steam client options."""
+
+    def __init__(self, entry: config_entries.ConfigEntry) -> None:
+        """Initialize options flow."""
+        self.entry = entry
+        self.options = dict(entry.options)
+
+    async def async_step_init(
+        self, user_input: dict[str, dict[str, str]] | None = None
+    ) -> FlowResult:
+        """Manage Steam options."""
+        if user_input is None:
+            try:
+                interface = await self.hass.async_add_executor_job(
+                    steam.api.interface, "ISteamUser"
+                )
+                friends = await self.hass.async_add_executor_job(
+                    ft.partial(
+                        interface.GetFriendList, steamid=self.entry.data[CONF_ACCOUNT]
+                    )
+                )
+                friends = (
+                    await self.hass.async_add_executor_job(friends.get, "friendslist")
+                )["friends"]
+                _users_str = str([user["steamid"] for user in friends])
+                names = await self.hass.async_add_executor_job(
+                    ft.partial(interface.GetPlayerSummaries, steamids=_users_str)
+                )
+                names = (await self.hass.async_add_executor_job(names.get, "response"))[
+                    "players"
+                ]["player"]
+                users = {
+                    name["steamid"]: {"name": name["personaname"], "enabled": False}
+                    for name in names
+                }
+            except steam.api.HTTPTimeoutError:
+                users = self.options[CONF_ACCOUNTS]
+            _users = users | self.options[CONF_ACCOUNTS]
+            self.options[CONF_ACCOUNTS] = {
+                k: v
+                for k, v in _users.items()
+                if k in users.keys() or self.options[CONF_ACCOUNTS][k]["enabled"]
+            }
+
+        else:
```

Comment:
Instead of an `else`, put a `return` at end of the if-statement

Reply:
If the if-statement has logic after this `else`,  then either move that logic up or invert the if-statement to do the `else` part first, and return then.